velox/exec/RowNumber.cpp

```diff
@@ -155,10 +164,12 @@ RowVectorPtr RowNumber::getOutputForSinglePartition() {
     numOutput = numInput;
   }
 
-  auto& rowNumbers = getOrCreateRowNumberVector(numOutput);
-
-  for (auto i = 0; i < numOutput; ++i) {
-    rowNumbers.set(i, ++numTotalInput_);
+  if (generateRowNumber_) {
+    FlatVector<int64_t>* rowNumbers =
```

Comment:
You can use "auto" since the type of rowNumbers is explicitly written in the casting.

Reply:
Yes. I had the same thought after the fact that I could use that.